manifests/d/Docker/DockerDesktop/4.0.0/Docker.DockerDesktop.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: Docker.DockerDesktop
+PackageVersion: 4.0.0
+PackageLocale: en-US
+Publisher: Docker Inc.
+PublisherUrl: https://www.docker.com
+PrivacyUrl: https://www.docker.com/legal/privacy
+Author: Docker Inc.
+PackageName: Docker Desktop
+PackageUrl: https://www.docker.com
+License: Apache License 2.0
```

Comment:
I've opened a similar PR #26422 earlier, also to fix the License and LicenseUrl.
The license was wrong, Docker Desktop never was Apache Licsene 2.0, that's only for eg. the Docker Engine which is part of Docker Desktop.

Reply:
Oof, that's not good. We'll have to correct that elsewhere.

I'll close this, thanks for letting me know!